index.html

```diff
@@ -630,16 +630,16 @@ <h2 class="large section-title">Notre plan de commandites</h2>
                             </div><!--pricing wrapper-->
                         </div><!--price col-->
                         <div class="col-md-4">
-                            <div class="pricing-wrapper wow animated fadeInRight" data-wow-delay=".9s">
+                            <div class="pricing-wrapper price-popular wow animated fadeInUp" data-wow-delay=".6s">
                                 <div class="pricing-wrapper-inner">
-                                    <h1>i5</h1>
-                                    <h2>1000 $</h2>
+                                    <h1>i7</h1>
```

Comment:
Thanks for the reordering. We should ask @Sytten to see if we could have another element for the i7. I don't like the fact that i5 has 5 elements and i7 has 4 and a `x` at the end. It's just about the presentation.